ui-components/src/locales/vi.json

```diff
@@ -1,749 +0,0 @@
-﻿{

```

Comment:
I don't think we need to delete this file or zh.json.

Reply:
You think removing these will cause merge conflicts, so it's best to keep the extra languages?

Reply:
Right, I don't see any harm in keeping them, especially since this is in the ui-components package.

Reply:
I think anything we can do to keep our ui-components closer to bloom is better, because Exygy and SFDS are working right now to solve some of the issues you mentioned in another comment and we would hopefully one day use ui-components via npm instead of having our own copy.

Reply:
Okay, done!